diff --color -ruN gismo-orig/src/gsUtils/gsSortedVector.h gismo-src/src/gsUtils/gsSortedVector.h
--- gismo-orig/src/gsUtils/gsSortedVector.h	2023-05-16 23:05:52
+++ gismo-src/src/gsUtils/gsSortedVector.h	2023-05-17 09:11:21
@@ -18,18 +18,18 @@
 //#include <algorithm>			// For lower_bound


-namespace std
-{
+// namespace std
+// {

-template<typename T1, typename T2>
-std::ostream& operator << ( std::ostream& os,
-    			const std::pair<T1,T2>& rhs )
-{
-    os << rhs.first << ", " << rhs.second;
-    return os;
-}
+// template<typename T1, typename T2>
+// std::ostream& operator << ( std::ostream& os,
+//     			const std::pair<T1,T2>& rhs )
+// {
+//     os << rhs.first << ", " << rhs.second;
+//     return os;
+// }

-}//namespace std
+// }//namespace std

 namespace gismo {
