diff --color -ruN gismo-orig/src/gsUtils/gsSortedVector.h gismo-src/src/gsUtils/gsSortedVector.h
--- gismo-orig/src/gsUtils/gsSortedVector.h	2023-11-13 21:40:55
+++ gismo-src/src/gsUtils/gsSortedVector.h	2023-11-13 21:41:44
@@ -17,7 +17,7 @@
 //#include <vector>
 //#include <algorithm>			// For lower_bound

-
+#if 0
 namespace std
 {

@@ -30,6 +30,6 @@
 }

 }//namespace std
-
+#endif
 namespace gismo {

 /** \brief This class is derived from std::vector, and adds sort tracking.
