diff --color -ruN uwebsockets-orig/src/AsyncSocket.h uwebsockets-src/src/AsyncSocket.h
--- uwebsockets-orig/src/AsyncSocket.h	2023-06-02 16:59:59
+++ uwebsockets-src/src/AsyncSocket.h	2023-06-02 17:01:35
@@ -201,9 +201,9 @@
         unsigned char *b = (unsigned char *) binary.data();

         if (binary.length() == 4) {
-            ipLength = sprintf(buf, "%u.%u.%u.%u", b[0], b[1], b[2], b[3]);
+            ipLength = snprintf(buf, 64, "%u.%u.%u.%u", b[0], b[1], b[2], b[3]);
         } else {
-            ipLength = sprintf(buf, "%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x",
+            ipLength = snprintf(buf, 64, "%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x",
                 b[0], b[1], b[2], b[3], b[4], b[5], b[6], b[7], b[8], b[9], b[10], b[11],
                 b[12], b[13], b[14], b[15]);
         }
